packages/fxa-content-server/app/styles/tailwind/images.css

```diff
@@ -0,0 +1,43 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+/* REACT NOTE: Should be able to refactor avatar stuff and remove this class,
+ * .with-default and other classes are added in `avatar-mixin.js`.
+ * Also what about the .default class in _avatar.scss? */
+.with-default {
+  @apply bg-center bg-no-repeat bg-contain;
+  background-image: inline('../images/default-profile.svg');
```

Comment:
We tend to [inline our SVGs](https://github.com/mozilla/fxa/tree/main/packages/fxa-settings#working-with-svgs) so I installed `postcss-assets` here to take our SVGs and inline them as BG images. There's pros and cons to it and while I believe the pros outweigh the cons, a con is that we don't utilize any caching and our CSS output is increased. However, every time content-server runs a build, we appended a hash to these image paths anyway to purposefully bust the cache, so we're not losing anything here by doing this except network requests 🤷‍♀️